deepmd/dpmodel/descriptor/make_base_descriptor.py

```diff
@@ -69,15 +69,13 @@ def distinguish_types(self) -> bool:
             """
             pass
 
-        @abstractmethod
         def compute_input_stats(self, merged):
```

Comment:
should be abstract method. the developer must provide the implementation of this interface.

Reply:
please not that the pt backend should preserve the same interface as the dpmodel.